src/dnaio/_core.pyx

```diff
@@ -62,10 +63,53 @@ cdef class Sequence:
     def __reduce__(self):
         return (Sequence, (self.name, self.sequence, self.qualities))
 
+    def qualities_as_bytes(self):
+        """Returns the qualities as a bytes object. This is a faster version
```

Comment:
I try to follow [PEP 257](https://www.python.org/dev/peps/pep-0257/) somewhat, which says this:
> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

Can you please change "Returns" to "Return" here and in the `fastq_bytes` function?